samples/apps/copilot-chat-app/webapi/Skills/DocumentQuerySkill.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Globalization;
+using Microsoft.SemanticKernel.Memory;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.SemanticFunctions.Partitioning;
+using Microsoft.SemanticKernel.SkillDefinition;
+
+namespace SemanticKernel.Service.Skills;
+
+/// <summary>
+///
+/// </summary>
+public class DocumentQuerySkill
```

Comment:
This looks to do both import and query. Consider renaming it to something like "DocumentMemorySkill".